packages/dashboard/src/constants/routes-config.ts

```diff
@@ -59,7 +201,7 @@ let routes_config: Array<TRoute>;
 const getRoutesConfig = ({ consumer_routes, is_deriv_crypto, Page404 }: TRoutesConfig): TRoute[] => {
     // For default page route if page/path is not found, must be kept at the end of routes_config array.
     if (!routes_config) {
-        const route_default = { component: Page404, getTitle: () => localize('Error 404') };
+        const route_default = { component: Page404, label: localize('Error 404') };
```

Comment:
can we stick with `getTitle` function? 
This is used for translation. When the language is changed, `getTitle` is called again to get the localized text.

Reply:
Sure, I replaced all labels with getTitle function